src/tests/ftest/util/daos_core_base.py

```diff
@@ -40,25 +40,15 @@ class DaosCoreBase(TestWithServers):
     :avocado: recursive
     """
 
-    TEST_PATH = "/run/daos_tests/Tests/*"
-
     def __init__(self, *args, **kwargs):
         """Initialize the DaosCoreBase object."""
         super(DaosCoreBase, self).__init__(*args, **kwargs)
-        self.subtest_name = None
 
-        test_timeout = self.params.get("test_timeout", self.TEST_PATH)
-        if test_timeout:
-            self.timeout = test_timeout
+        self.num_clients = None
+        self.scalable_ep = None
```

Comment:
As we discussed all of the changes to this file should be reverted and then this definition should be added before `test_timeout = self.params.get("test_timeout", self.TEST_PATH)` is called:
```
        self.TEST_PATH = "/run/daos_tests/Tests/{}".format(self.get_test_name())
```